ttab/loads/define_model.py

```diff
@@ -3,6 +3,7 @@
 
 import timm
 import torch
+from ttab.loads.models.wideresnet_robust_bench import WideResNetRB
```

Comment:
It's good to test the correctness of implementation using the same model as the paper does, but I think there is no need to integrate this particular model into our benchmark, especially when it has to build another independent model class.